pkg/render/node.go

```diff
@@ -534,9 +534,6 @@ func (c *nodeComponent) nodeRole() *rbacv1.ClusterRole {
 					"licensekeys",
 					"packetcaptures",
 					"remoteclusterconfigurations",
-					"stagedglobalnetworkpolicies",
-					"stagedkubernetesnetworkpolicies",
-					"stagednetworkpolicies",
```

Comment:
Node already has access to these resource here: https://github.com/tigera/operator/blob/a26c3147e37f1c3632f600b19396956b437b53f1/pkg/render/node.go#L450-L452